core/static/core/tags/timesheet.tag

```diff
@@ -0,0 +1,45 @@
+<timesheet>
+    <td if={ !edit }>{ name }</td>
+    <td if={ edit }><input type="text" class="form-control form-control-sm" value="{ name }" onkeypress="return event.keyCode != 13;">
+    <td class="text-right">
+        <a if={ !edit } class="btn btn-warning btn-sm" onclick={ editTimesheet }>Edit</a>
+        <a if={ edit } class="btn btn-success btn-sm" onclick={ saveTimesheet }>Save</a>
+        <a class="btn btn-primary btn-sm" onclick={ goToTasks }>Tasks</a>
+        <a class="btn btn-primary btn-sm" onclick={ goToEntries }>Entries</a>
+    </td>
+
+
+    <script>
+        var self = this;
+        var edit = false;
+
+
+        editTimesheet(e) {
+            self.edit = true;
+            self.update();
+        }
+
+
+        goToTasks(e) {
+            document.location.href = tasksUrl + e.item.id;
+        }
+
+
+        goToEntries(e) {
+            document.location.href = entriesUrl + e.item.id;
+        }
+
+
+        saveTimesheet(e) {
+            e.preventDefault();
+            let tr = self.root;
+            let td = $(tr).find('td');
+            self.name = $(td[0]).find('input').val();
```

Comment:
@cdubz yep, add a "ref" attribute to the input. You can then call it with self.refs.<nameOfRef>.value